lib/util.js

```diff
@@ -371,10 +371,11 @@ function formatValue(ctx, value, recurseTimes) {
   // Look up the keys of the object.
   var keys = Object.keys(value);
   var visibleKeys = arrayToHash(keys);
+  var symbolKeys = Object.getOwnPropertySymbols(value);
```

Comment:
I just don't know the reasons for not listing non-enumerable properties. For the sake of explaining my experience with the matter, I'll testify that I:
1. never made a property non-enumerable before examining this issue
1. never had to examine the non-enumerable properties of an object. At least not that I know of.

So, it may help that one with experience with the topic of enumerability and non-enumerability will share his thoughts. Also, the thoughts of one who knows the intention of symbols in this regard and why this is the behavior of `Object.getOwnPropertySymbols`.